quelpa.el

```diff
@@ -6,8 +6,8 @@
 ;; Author: steckerhalter
 ;; URL: https://github.com/quelpa/quelpa
 ;; Version: 0.0.1
-;; Package-Requires: ((emacs "24.3"))
-;; Keywords: package management build source elpa
+;; Package-Requires: ((emacs "25.1"))
+;; Keywords: tools
```

Comment:
keywords probably should be kept the same (or adding additionaly)

Reply:
Done 7e54f8b.
I was misunderstanding package-lint warning ``You should include standard keywords: see the variable `finder-known-keywords'``.